docs/package.json

```diff
@@ -1,10 +1,11 @@
 {
   "private": true,
   "scripts": {
-    "dev": "run-p dev:*",
+    "dev": "yarn build:index-docs && run-p dev:*",
     "dev:routify": "cross-env NODE_ENV=development routify run",
     "dev:svite": "vite dev",
     "build": "run-s build:*",
+    "build:index-docs": "node scripts/index-docs.js",
```

Comment:
I don't think the generated search index should be checked into source control; as a result, we need to build the index before dev/build modes.